paperqa/agents/tools.py

```diff
@@ -399,6 +400,253 @@ async def complete(
         return f"{'Success' if has_successful_answer else 'Unsure'} | {state.status}"
 
 
+class ClinicalTrialsSearch(NamedTool):
+    TOOL_FN_NAME = "clinical_trials_search"
+
+    model_config = ConfigDict(extra="forbid", arbitrary_types_allowed=True)
```

Comment:
Why `arbitrary_types_allowed=True`? I don't think we need it (may be wrong here tho)